docs/incremental-xmlupload.md

```diff
@@ -85,3 +86,28 @@ dsp-tools xmlupload --incremental additional_data_replaced_[timestamp].xml
 | <center>Important</center>                                                                                                                      |
 |-------------------------------------------------------------------------------------------------------------------------------------------------|
 | Internal IDs and IRIs cannot be mixed within the same file. An XML file uploaded with the incremental option must not contain any internal IDs. |
+
+
+
+## 4. Continue an interruped xmlupload
+
+If an xmlupload didn't finish successfully, 
+some resources have already been created, but others not.
+If the remaining resources have references to created ones,
+these references must be made with IRIs,
+and the created resources must be removed from the XML file
+(otherwise they would be created a second time).
+
+In such a case, proceed as follows:
+
+```bash
+dsp-tools xmlupload data.xml
+# crash: some resources have been uploaded, and a id2iri_mapping_[timestamp].json file has been written
+# fix the reason for the crash
+
+# replace the IDs and remove the created resources with:
+dsp-tools id2iri data.xml --remove-resources id2iri_mapping_[timestamp].json
+
+# upload the outputted XML file with
+dsp-tools xmlupload data_replaced_[timestamp].xml
+```
```

Comment:
I would remove the # comments and make those normal text. Because this is not stuff you type in.

Reply:
Done